manual/manual/tutorials/moduleexamples.etex

```diff
@@ -178,19 +178,12 @@ module type PRIOQUEUE_WITH_OPT =
 \section{Functors}
 \pdfsection{Functors}
 
-Functors are ``functions'' from structures to structures. They are used to
-express parameterized structures: a structure \var{A} parameterized by a
-structure \var{B} is simply a functor \var{F} with a formal parameter
-\var{B} (along with the expected signature for \var{B}) which returns
-the actual structure \var{A} itself. The functor \var{F} can then be
-applied to one or several implementations \nth{B}{1} \ldots \nth{B}{n}
-of \var{B}, yielding the corresponding structures
-\nth{A}{1} \ldots \nth{A}{n}.
+Functors are ``functions'' from modules to modules. Functors
+let you create parameterized modules and then provide other modules as parameter(s)
+to get a specific implementation.  For instance, a "Set" module implementing sets
+as sorted lists could be parameterized to work with any module that provides a type
+and a comparison method "compare" (such as "OrderedString"):
```

Comment:
I would use `comparison function` rather than `method` to avoid a potential confusion with object methods.